src/operation/raw_output.rs

```diff
@@ -0,0 +1,77 @@
+use crate::{
+    cmap::{Command, RawCommandResponse, StreamDescription},
+    error::Result,
+};
+
+use super::Operation;
+
+pub(crate) struct RawOutput<Op>(pub(crate) Op);
```

Comment:
Can we add a docstring indicating this type's purpose? If I'm understanding it right, it forwards all implementation to the underlying op except that it performs no validation or parsing of the response, is that right?

Reply:
Yup, that's correct - added a comment.

Reply:
Whoops, fixed.